recipes-wpe/wpeframework/include/wpeframework-ocdm-playready.inc

```diff
@@ -0,0 +1,17 @@
+SUMMARY = "WPE Framework OpenCDMi module for playready"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=66fe57b27abb01505f399ce4405cfea0"
+
+require wpeframework-plugins.inc
+
+DEPENDS += " playready"
+
+SRC_URI = "git://git@github.com/rdkcentral/OCDM-Playready.git;protocol=https;branch=${SRCBRANCH}"
```

Comment:
Could you please SRC_URI in each .bb file instead of using additional variable?

Reply:
done